styles/duckduckgo/catppuccin.user.css

```diff
@@ -297,6 +297,12 @@
       background: linear-gradient(rgba(40, 40, 40, 0), @surface0);
     }
 
+    /* popup */
+    .atb-new .badge-link__checkbox.badge-link__checkbox_checked {
+        background: @mantle;
+        color: @text;
+    }
```

Comment:
Are you sure there aren't variables we can use here? I'm seeing other already themed variables with `atb` in them which makes me think we could use vars here.

Reply:
i didnt check to see if there was any variables which we can use, maybe you can theme these ``atb`` variables and check to see if they actually theme?

Reply:
Yes... I'm asking you to do that. You should *always* be checking for variables.

Reply:
i checked the variables and theres no way we can use them here so this codes good